config/database.yml.example

```diff
@@ -1,88 +1,88 @@
-# PostgreSQL. Versions 9.3 and up are supported.
-#
-# Install the pg driver:
-#   gem install pg
-# On macOS with Homebrew:
-#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
```

Comment:
this change doesn't make any sense

Reply:
how i can fix it?
I didn't change these files.
When I set up my local database, I deleted them. Then added it back. But the git decided to comment them out as modified.